ui/src/app/customizations/tech-extensions/tech-extensions-list.component.ts

```diff
@@ -26,6 +26,11 @@ export class TechExtensionsListComponent implements OnInit {
   listConfig: ListConfig;
   @ViewChild(TechExtensionDeleteModalComponent)
   deleteModal: TechExtensionDeleteModalComponent;
+  itemUseMapping: { [valueComparator: string]: string } = {
+    '=0': '<strong>0</strong> integrations',
+    '=1': '<strong>1</strong> integrations',
```

Comment:
Probs you want the output to be `<strong>1</strong> integration` and not `integrations` (please note the 's')